pyproject.toml

```
@@ -55,19 +55,18 @@ docs = [
   "sphinx-gallery",
 ]
 
+[project.scripts]
+pybamm_cookiecutter = "pybamm_cookiecutter.cli:pybamm_cookiecutter_cli"
+
+[project.entry-points."pipx.run"]
+pybamm_cookiecutter = "pybamm_cookiecutter.cli:pybamm_cookiecutter_cli"
+
```

```suggestion
[project.scripts]
pybamm-cookiecutter = "pybamm_cookiecutter.cli:pybamm_cookiecutter_cli"

[project.entry-points."pipx.run"]
pybamm-cookiecutter = "pybamm_cookiecutter.cli:pybamm_cookiecutter_cli"

```

I think these should both be `pybamm-cookiecutter`, similar to how other packages that offer CLIs with hyphens in their names have been packaged and made available through `pipx` – hyphens are accepted in this case, and having an underscore doesn't seem necessary.